avm/res/elastic-san/elastic-san/tests/e2e/pe/main.test.bicep

```diff
@@ -0,0 +1,96 @@
+targetScope = 'subscription'
+
+metadata name = 'Private endpoint-enabled deployment'
+metadata description = 'This instance deploys the module with private endpoints.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@sys.description('Optional. The name of the resource group to deploy for testing purposes.')
+@sys.maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-microsoft.elasticsan-${serviceShort}-rg'
+
+@sys.description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@sys.description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'esanpe'
+
+@sys.description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
+  params: {
+    virtualNetworkName: 'dep-${namePrefix}-vnet-${serviceShort}'
+    location: resourceLocation
+  }
+}
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+var tags = {
```

Comment:
Though used twice and hence justifying a variable I'd recommend to move them into the PE deployment below to have it also rendered in the usage example in the readme.

Reply:
Make sense - Tags moved to the code to be documented as part of the example. I'd like to keep tags in this PE test because we are sending tags further to the Private Endpoints and we are testing tags assignments as part of the test.